pkg/controller/endpointslice/endpointslice_controller_test.go

```diff
@@ -329,16 +332,18 @@ func TestOnEndpointSliceUpdate(t *testing.T) {
 
 // Ensure SyncService handles a variety of protocols and IPs appropriately.
 func TestSyncServiceFull(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.EndpointSliceTerminatingCondition, true)()
```

Comment:
Same comment here, I hate to lose test coverage of existing/default functionality. Until this feature gate is enabled by default I'd prefer to have any test coverage be in addition to existing test coverage.